package.json

```diff
@@ -8,15 +8,16 @@
     "dev": "ts-node-dev --respawn --transpile-only src/server.ts",
     "build": "tsc",
     "start": "node dist/server.js",
-    "migrate": "npx prisma migrate dev --schema=src/prisma/schema.prisma",
-    "generate": "prisma generate --schema=src/prisma/schema.prisma",
-    "deploy": "npx prisma migrate deploy --schema=src/prisma/schema.prisma",
+    "migrate": "npx prisma migrate dev",
+    "generate": "prisma generate",
+    "deploy": "npx prisma migrate deploy",
     "studio": "prisma studio --schema=src/prisma/schema.prisma",
     "reset": "npx prisma migrate reset --force --schema=src/prisma/schema.prisma",
     "seed": "npx prisma db seed --schema=src/prisma/schema.prisma",
-    "lint": "biome check .",
-    "lint-fix": "biome check --fix --unsafe .",
-    "format": "biome  check --write ."
+    "lint": "biome check --changed --no-errors-on-unmatched",
+    "lint-fix": "biome check --fix --unsafe --staged --no-errors-on-unmatched",
+    "format": "biome check --write --staged --no-errors-on-unmatched",
```

Comment:
使用 `--changed` 或是 `--staged` 可以不用每個檔案都跑，只要關注有變動的。

`--no-errors-on-unmatched`："Silence errors that would be emitted in case no files were processed during the execution of the command." - 忽略未匹配的文件或路徑，而不將其視為錯誤。